library/LuaApi.cpp

```diff
@@ -2456,6 +2456,17 @@ static int screen_findGraphicsTile(lua_State *L)
     }
 }
 
+static int screen_raise(lua_State *L) {
```

Comment:
Do we need the custom wrapper here? `raise()` takes a `df::viewscreen*` only and returns `void`, so it ought to be natively wrappable with `WRAPM` and should behave the same way, I think

Reply:
The call to `dfhack_lua_viewscreen::get_pointer()` to convert from the Lua Screen table to the screen._native pointer does not seem to be automatic. At least when I tried a standard `WRAPM`, it built, but it didn't function.

Reply:
Oh, I missed the `lua` part of the class name in `dfhack_lua_viewscreen::get_pointer()`. Disregard.